frame/elections-phragmen/src/lib.rs

```diff
@@ -1295,8 +1307,21 @@ mod tests {
 		type LoserCandidate = ();
 		type KickedMember = ();
 		type WeightInfo = ();
-		type MaxVoters = PhragmenMaxVoters;
-		type MaxCandidates = PhragmenMaxCandidates;
+		type MaxVoters = MaxVoters;
+		type MaxCandidates = MaxCandidates;
+		type ElectionSolver = SequentialPhragmen<Self::AccountId, Perbill>;
+		type SolverWeightInfo = NposWeightInfo;
+	}
+
+	pub struct NposWeightInfo;
+	impl frame_election_provider_support::WeightInfo for NposWeightInfo {
```

Comment:
Don't we have any default impl for this? also, perhaps we can check for the weight values in the tests as well.